its/ruling/src/integrationTest/resources/sources/kubernetes/S2260/parsing-error.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+spec: a #b
+  c
+  d
```

Comment:
Is this a valid YAML? Shouldn't `c` and `d` be either keys or array elements? Or is parsing error caused by something else?

Reply:
It's not a valid YAML, but it's here on purpose to have confirmation for parsing error. See filename and location.